Jenkinsfile-build-and-deploy

```diff
@@ -46,6 +46,11 @@ node('linux'){
                        }
                     } else {
                         stage('Build developement image') {
+                            if (params.crowdin_branch != null) {
+                                sh """#!/bin/sh
+                                ./tasks/download-translations.sh /etc/crowdin/crowdin.conf ${params.crowdin_branch}"""
```

Comment:
I assume that built image use that translations, so where are they stored?

Reply:
Translations are stored in the Crowdin third-party service and are downloaded using `i18n-tool` (also called `crowdin` :) ). The tool needs `*.conf` files from `crowdin` folder in the repo, then translations are precompiled to `json` files.

I hope this answer your question

Reply:
k, thx!